### ISSUE TITLE
Straw bale loading sometimes visually fails

### ISSUE THREAD

**Issue by hvdklauw** [loki79uk/FS25_UniversalAutoload#46]
Version: 0.0.0.7
Trailer: Farmtech DPW 1800
Config for the trailer:
```
        <vehicle configFileName="data/vehicles/farmtech/dpw1800/dpw1800.xml">
            <configuration selectedConfigs="ALL">
                <loadingArea offset="-0.036 1.130 -0.820" width="2.931978" length="9.728788" height="2.403885"/>
            </configuration>
        </vehicle>
```

Steps:

- Stop next to straw bales 
- Press Shift+r to load them on
- Sometimes they appear on the trailer, sometimes they stay on the ground
- Pressing Shift + I to unload does move the bale that I tried to load to the other side in the unload zone, but not neatly stacked with the rest

Last bit of the log file:
```
2024-12-01 21:00 *** UNIVERSAL AUTOLOAD - FOUND NEW OBJECT TYPE: C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d ***
2024-12-01 21:00 Round Bale
2024-12-01 21:00   width: 1.2
2024-12-01 21:00   height: 1.25
2024-12-01 21:00   length: 1.25
2024-12-01 21:00 rootNode: 20003732
2024-12-01 21:00   size X: 1.3097991943359375
2024-12-01 21:00   size Y: 1.31011962890625
2024-12-01 21:00   size Z: 1.18328857421875
2024-12-01 21:00   offset X: 0.00000762939453125
2024-12-01 21:00   offset Y: -0.000091552734375
2024-12-01 21:00   offset Z: -0.000030517578125
2024-12-01 21:00 Round Bale flipYZ
2024-12-01 21:00   >> C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d [1.310, 1.183, 1.310] - BALE
2024-12-01 21:00 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:00 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:01 Super Strength Activated!
2024-12-01 21:01 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:01 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:01 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:01 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:01 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:01 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 FULLY UNLOADED...
2024-12-01 21:02 CALLBACK actionEventToggleLoading: Farmtech DPW 1800 - UAL #20462669
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
2024-12-01 21:02 [20462669] FIND LOADING PLACE FOR C:/Users/hvdkl/Documents/My Games/FarmingSimulator2025/mods/FS25_RounderWrappedRoundbales/assets/roundbale125/roundbale125.i3d
```

Have 6 bales on the trailer and the seventh was causing the issue.

Edit: Did another unload and load and then it picked it up correctly...
Edit2: Also tried to pick up the offending bale with superstrength to see if moving it helped, but it didn't

**Comment by loki79uk** [comment]
Is this on MP?  If so, then the next release will fix this issue (should be in the next hour or two)

**Comment by loki79uk** [comment]
This should be fixed now in v0.0.0.8: https://github.com/loki79uk/FS25_UniversalAutoload/releases

**Comment by hvdklauw** [comment]
Upgraded to 0.0.0.9,  now it works
